modules/cluster-logging-release-notes-5.0.4.adoc

```diff
@@ -0,0 +1,23 @@
+[id="cluster-logging-release-notes-5-0-4"]
+= OpenShift Logging 5.0.4
+
+This release includes link:https://access.redhat.com/errata/RHSA-2021:2136[RHSA-2021:2136 Moderate: Openshift Logging security and bugs update (5.0.4)]
```

Comment:
Are you not putting a period at the end of this sentence because it ends with a link? If so, that makes sense, and I see you're consistent without putting periods at the end of a link throughout the doc. I'm only asking because I see this is a full sentence. Either way (with or without a period) works fine!

Reply:
Good catch, Jake. Every sentence should end with a period, period. :-)